src/plugins/visible-range/index.js

```diff
@@ -30,13 +31,16 @@ export default class VisibleRangePlugin extends AbstractPlugin {
     this.config = {
       latitudeRange : null,
       longitudeRange: null,
+      usePanoData: false,
     };
 
     if (options) {
+      this.config.usePanoData = !!options.usePanoData;
       this.setLatitudeRange(options.latitudeRange);
       this.setLongitudeRange(options.longitudeRange);
     }
 
+    this.psv.on(CONSTANTS.EVENTS.PANORAMA_LOADED, this);
```

Comment:
I decided to always enable this handler, not only when `usePanoData` is `true`. That makes it possible for users to load a panorama without `usePanoData` first, then change the plugin config and enable it in a new panorama they load in the same viewer. Not sure if this is really a use case you want to support.

However, it's not possible to enable `usePanoData` on the same panorama after creating a viewer with it disabled. Not without copying the functionality of the getPano*Range` functions anyway. I think this would require exposing an extra function, either the `getPano*Range` functions or some sort of `enableUsingPanoData` function. I don't need this functionality.